File: 2-semestre/uri-online-judge/java/1115.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException { 
        int x, y;

        Scanner entrada = new Scanner(System.in);

        do {
            x = entrada.nextInt();
            y = entrada.nextInt();

            if (x == 0 || y == 0) break;
            else if (x > 0 && y > 0) System.out.printf("primeiro\n");
            else if (x < 0 && y > 0) System.out.printf("segundo\n");
            else if (x < 0 && y < 0) System.out.printf("terceiro\n");
            else System.out.printf("quarto\n");
        } while (x != 0 && y != 0);
    }
}

File: 2-semestre/uri-online-judge/javascript/1115.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

do {
    let linha = lines.shift().split(" ");
    x = parseInt(linha[0]);
    y = parseInt(linha[1]);

    if (x === 0 || y === 0) break;
    else if (x > 0 && y > 0) console.log("primeiro");
    else if (x < 0 && y > 0) console.log("segundo");
    else if (x < 0 && y < 0) console.log("terceiro");
    else console.log("quarto");
} while (x !== 0 && y !== 0);
